cmd/system-probe/modules/process_tracer.go

```diff
@@ -0,0 +1,89 @@
+package modules
+
+import (
+	"errors"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sync/atomic"
+	"time"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/utils"
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+	"github.com/DataDog/datadog-agent/pkg/process/procutil"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// ErrProcessTracerUnsupported is an error type indicating that the process_tracer is not support in the running environment
+var ErrProcessTracerUnsupported = errors.New("process_tracer module unsupported")
+
+// ReturnZeroStats controls whether process_tracer returns stats that are all zeros.
+// Currently it's set to false to reduce the deserialization work process-agent has to do
+var ReturnZeroStats = false
+
+// ProcessTracer is a module that fetches process level data
+var ProcessTracer = api.Factory{
+	Name: "process_tracer",
+	Fn: func(agentConfig *config.AgentConfig) (api.Module, error) {
+		if !agentConfig.CheckIsEnabled("Process Tracer") {
+			log.Infof("Process tracer disabled")
+			return nil, api.ErrNotEnabled
+		}
+
+		log.Infof("Creating process_tracer for: %s", filepath.Base(os.Args[0]))
+
+		p := procutil.NewProcessProbe()
+		// currently this is true for all environments except linux
```

Comment:
Nit: this type of comment gets stale very quickly—I think it can just be removed

Reply:
Will remove, it's indeed unnecessary.